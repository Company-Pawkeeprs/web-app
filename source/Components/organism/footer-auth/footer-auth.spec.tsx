import { cleanup, render, screen } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import FooterAuth from './footer-auth';

afterEach(cleanup)

describe('FooterAuth', () => {
    it('renders', async () => {
        const { baseElement } = render(<FooterAuth />);
        expect(baseElement).toMatchSnapshot();
    });
    it('should ensure the footer contains the PawKeeprs text', async () => {
        render(<FooterAuth />);
        // Verifica se PawKeeprs est√° no footer
        expect(screen.getByText(/PawKeeprs/i)).toBeTruthy();
    })
})