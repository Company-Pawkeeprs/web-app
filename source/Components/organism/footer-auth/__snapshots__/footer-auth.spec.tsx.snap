// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FooterAuth > renders 1`] = `
<body>
  <div>
    <footer
      class="footer"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-lg-12"
          >
            <div
              class="text-center"
            >
              <p
                class="mb-0"
              >
                Â© 
                2023
                 PawKeeprs. 
              </p>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>
</body>
`;
